packages/flutter_tools/lib/src/android/android_workflow.dart

```diff
@@ -28,7 +28,8 @@ class AndroidWorkflow extends Workflow {
     androidValidator.addValidator(new Validator(
       'Android SDK',
```

Comment:
This says SDK, but the web page we direct them to says "Android Studio". This is confusing.